syft/core/workers/socket.py

```diff
@@ -124,6 +125,9 @@ def __init__(self,  hook=None, hostname='localhost', port=8110, max_connections=
             else:
                 print("Ready!")
 
+    def whoami(self):
+        return json.dumps({"hostname": self.hostname, "port": self.port, "id": self.id})
```

Comment:
This is really useful for sharing your id with other worker. Do we want to have the ability to call this on any known worker so that workers can share the id of workers that they know with other workers they know? Or more specifically the client can tell the workers who the other workers are

Reply:
Yeah! Seems like we should. I'll add a default to BaseWorker.